src/Symfony/Component/Validator/Tests/Constraints/UuidValidatorTest.php

```diff
@@ -263,4 +263,36 @@ public function testInvalidNonStrictUuidNamed()
             ->setCode(Uuid::INVALID_CHARACTERS_ERROR)
             ->assertRaised();
     }
+
+    /**
+     * @dataProvider getUuidForTimeBasedAssertions
+     */
+    public function testTimeBasedUuid(string $uid, bool $expectedTimeBased)
+    {
+        $constraint = new Uuid([
+            'versions' => Uuid::TIME_BASED_VERSIONS,
+        ]);
+
+        $this->validator->validate($uid, $constraint);
+
+        if ($expectedTimeBased) {
+            $this->assertNoViolation();
+        } else {
+            $this->buildViolation('This is not a valid UUID.')
+                ->setParameter('{{ value }}', '"'.$uid.'"')
+                ->setCode(Uuid::INVALID_TIME_BASED_VERSION_ERROR)
+                ->assertRaised();
+        }
+    }
+
+    public function getUuidForTimeBasedAssertions()
```

Comment:
```suggestion
    public static function getUuidForTimeBasedAssertions()
```

From PHPUnit 10 on, non static data providers are deprecated

Reply:
Oh alright! Updated :+1:

Reply:
Thank you!